### PR TITLE
Removes underscore from name of greater than or equal circuit

### PR DESCRIPTION
**Pull request by 1fbff5f83b23d39d38b1dfcb4cac8d9b**

### CODE REVIEW

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
you tried bot

**Comment by optimumtact** [comment]
it's cause you said remove in the PR title :grin:

File: code/modules/integrated_electronics/subtypes/logic.dm [hunk 1]
```diff
@@ -212,6 +212,10 @@
 	if(comparable(A, B))
 		return A.data >= B.data
 
+/obj/item/integrated_circuit/logic/binary/greater_than_or_equal/old //for backwards compatibility
+	name = "greater_than or equal gate"
+	spawn_flags = 0
```
Comment (ninjanomnom) [inline code/modules/integrated_electronics/subtypes/logic.dm]: I really don't like this "add another typepath for backwards compat" thing

Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [reply code/modules/integrated_electronics/subtypes/logic.dm]: It lets old circuits work without affecting new ones, I don't think it's a big deal

Comment (ninjanomnom) [reply code/modules/integrated_electronics/subtypes/logic.dm]: Many "not a big deal"s turn into an unmaintainable mess

Comment (PKPenguin321) [reply code/modules/integrated_electronics/subtypes/logic.dm]: It would be really easy to add an out of game script to update old circuit pastes

Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [reply code/modules/integrated_electronics/subtypes/logic.dm]: I don't really see how it makes things difficult to maintain

Comment (Cyberboss) [reply code/modules/integrated_electronics/subtypes/logic.dm]: Typepath bloat is a real thing, just do a remapping list in the decoder

Comment (ExcessiveUseOfCobblestone) [reply code/modules/integrated_electronics/subtypes/logic.dm]: you're not actually fixing the issue if you knowingly put the issue as a subtype to coddlepeople who don't know find/replace.

**Pull request merged**